<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Анализатор плагиата</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabAnalysis">
    <attribute name="title">
     <string>Анализ текста</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>611</width>
       <height>361</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Введите текст для проверки...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCheck">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>360</y>
       <width>291</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
        background-color: #4CAF50;
        color: white;
        border: none;
        border-radius: 5px;
        font-weight: bold;
      }
      QPushButton:hover {
        background-color: #45a049;
      }</string>
     </property>
     <property name="text">
      <string>Проверить на плагиат</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>40</y>
       <width>291</width>
       <height>301</height>
      </rect>
     </property>
     <property name="title">
      <string>Результаты анализа</string>
     </property>
     <widget class="QLabel" name="lblUniqueness">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Уникальность: 0%</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="lblChars">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Всего символов: 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblNoSpaces">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Без пробелов: 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblWords">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>251</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Количество слов: 0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSave">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>251</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить отчет</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Текст для проверки:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabSettings">
    <attribute name="title">
     <string>Настройки</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>891</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Параметры анализа</string>
     </property>
     <widget class="QCheckBox" name="chkIgnoreCase">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Игнорировать регистр букв</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSaveSettings">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>120</y>
        <width>171</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить настройки</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
